experiments/config.toml

```diff
@@ -0,0 +1,74 @@
+title = "Mspush-Java's experiments."
+
+[owner]
+name = "David Ferreira Quaresma"
+email = "davidferreiraquaresma@gmail.com"
+
+[setup]
+num_vms = 4
+vcpus = 2
+ram_gb = 4
+java_version = "10.0.1"
+gcc="5.4.0" # compiled the libgc.so
+maven="3.3.9" # build all java project.
+go_version = "go1.10 linux/amd64"  # compiled the gci proxy binary.
+nginx_version = "nginx/1.14.0 (Ubuntu)"
+so_version = "Ubuntu 18.04 LTS"
+kernel_version = "4.15.0-29-generic"
+
+[execution]
+# exports: export commands which should be used to export common environment variables. Should use outdir=stateless for stateless experiments and outdir=stateful for stateful experiments.
+exports = 'export outdir="stateless"; export expdir=${PWD}'
+
+# killall: kill all processes started by the experiment.
+killall = 'for ip in $ip1 $ip2 $ip3 $ip4; do ssh -i ${sshkey} ubuntu@${ip} "killall java 2>/dev/null; rm gc.log shed_${expid}_${ip}.csv 2>/dev/null; killall mon.sh 2>/dev/null; killall gci-proxy 2>/dev/null; rm proxy.* 2>/dev/null;"; done'
```

Comment:
killall should not do file cleanups if it is strictly necessary.

> rm gc.log shed_${expid}_${ip}.csv 2>/dev/null;
> rm proxy.* 2>/dev/null;

Reply:
Done